graphql/util.go

```diff
@@ -235,6 +235,9 @@ func SchedulePatch(ctx context.Context, patchId string, version *model.Version,
 	if err = p.SetDescription(patchUpdateReq.Description); err != nil {
 		return errors.Wrap(err, "Error setting description"), http.StatusInternalServerError, "", ""
 	}
+	if err = model.UpdateVersionMessage(p.Version, patchUpdateReq.Description); err != nil {
```

Comment:
We should only do this if p.Version != "" (so should move to 266)

Reply:
done